pyproject.toml

```diff
@@ -31,7 +31,7 @@ license = {text = "BSD 3-clause"}
 requires-python = ">=3.8,<4"
 dependencies = [
     "cloudpickle >= 1.5.0",
-    "holidays >= 0.13",
+    "holidays < 0.33",
```

Comment:
I think we need to still include `>=0.13` here as older versions might be problematic.

`"holidays >= 0.13, <0.33"`

Reply:
ah, makes sense. I'll update.